### PR TITLE
minor: Removed missed and redundant supressions

### PR DESCRIPTION
**Pull request by shashwatj07**

### CODE REVIEW

File: config/checkstyle_input_suppressions.xml [hunk 1]
```diff
@@ -21,41 +21,7 @@
...
-            files="annotation[\\/]missingdeprecated[\\/]InputMissingDeprecatedClass.java"/>
-
-  <suppress id="ConfigCommentOnTopInputs"
-            files="annotation[\\/]missingdeprecated[\\/]InputMissingDeprecatedGood.java"/>
-
-  <suppress id="ConfigCommentOnTopInputs"
-            files="annotation[\\/]missingdeprecated[\\/]InputMissingDeprecatedMethod.java"/>
-
-  <suppress id="ConfigCommentOnTopInputs"
-            files="annotation[\\/]missingdeprecated[\\/]InputMissingDeprecatedSingleComment.java"/>
```
Comment (rnveach) [inline config/checkstyle_input_suppressions.xml:24]: triple blank line

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/InputPackageAnnotation2.java [hunk 2]
```diff
@@ -9,6 +9,6 @@
 @Deprecated
 package com.puppycrawl.tools.checkstyle.checks.annotation.packageannotation;
 
-public class InputPackageAnnotation2 {
+public class InputPackageAnnotation2 { // ok
```
Comment (timurt) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/InputPackageAnnotation2.java:12]: could you please explain why `// ok` comments were added?

Comment (shashwatj07) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/InputPackageAnnotation2.java]: It was a slight mistake, most of them are actually files with no violations. However, a couple of them were mistaken.
romani said we don't care about violation comments right now, we'll place them during migration.

Comment (shashwatj07) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/InputPackageAnnotation2.java]: I've fixed these, however

File: config/checkstyle_input_suppressions.xml (hunk 1 above)
Comment (shashwatj07) [reply config/checkstyle_input_suppressions.xml:24]: removed extra blank lines

**Review by timurt — APPROVED** [review]

**Pull request merged**